src/webhooks/webhooks.service.ts

```diff
@@ -17,6 +18,9 @@ export class WebhooksService {
     const userDetails = await this.userRepository.findOne({ email: client_email });
 
     if (!userDetails) {
+      await axios.post(`${process.env.SLACK_WEBHOOK_URL}`, {
+        text: `${client_email} doest not exist in the bloom backend`,
```

Comment:
Can we make this text
`🚨 Unknown email address made a therapy booking - ${client_email}`

Reply:
it should be sent to the `bloom-therapy` channel in slack! I saw there was a bloom-backend integration added to `platform-build` channel, but can't see an example of one of these messages, so has it been tested at all?